.pre-commit-config.yaml

```diff
@@ -12,6 +12,7 @@ repos:
       ^
         .*
       $
+    stages: ["manual"]
```

Comment:
This is rather pointless since everything is excluded already.

Reply:
Request to do it like this came from @cidrblock 

We will move them out of manual one by one using reviews that involve entire team and use the new label "quorum-required". It is about getting approval for any new rule. It will require to fix exceptions/exclusions before doing that or at the same time. Basically anything optional is manual and rub by the lint-vetting non-voting job.

Reply:
This also does has an inconsistent style with the rest of the places in the file.